feature-libs/organization/administration/components/shared/list/list.component.html

```diff
@@ -22,19 +22,25 @@ <h3>
         </div>
 
         <div class="actions">
-          <ng-select
-            class="sort"
-            *ngIf="data.pagination?.sort"
-            [searchable]="false"
-            [clearable]="false"
-            (change)="sort(data.pagination)"
-            [tabIndex]="0"
-            [(ngModel)]="sortCode"
```

Comment:
missing `[attr.aria-label]="selectedLabel || placeholder"` to match with `ng-select` used in `cx-sorting` component